lib/docs/filters/padrino/entries.rb

```diff
@@ -0,0 +1,17 @@
+module Docs
+  class Padrino
+    class EntriesFilter < Docs::EntriesFilter
+      def get_name
+        node = at_css('h1')
+        result = node.content.strip
+        result << ' event' if type == 'Events'
+        result << '()' if node['class'].try(:include?, 'function')
+        result
+      end
+
+      def get_type
+        return 'Ruby'
```

Comment:
The type is used to categorize entries in the sidebar:

![devdocs - nokogiri documentation 2016-06-04 12-21-52](https://cloud.githubusercontent.com/assets/17579/15800718/0f360d42-2a4f-11e6-85d0-778b9b91eeeb.png)

We need to return different ones based on the class, module, etc.  For example:

https://github.com/Thibaut/devdocs/blob/ebd6a2204da031328c1ba707e9d119e3969b4146/lib/docs/filters/nokogiri2/entries.rb#L4-L8